src/Integration/NewConnectedMode/UnbindCommand.cs

```diff
@@ -0,0 +1,84 @@
+/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using System.Diagnostics;
+using Microsoft.VisualStudio;
+using Microsoft.VisualStudio.Shell;
+using SonarLint.VisualStudio.Integration.Resources;
+
+namespace SonarLint.VisualStudio.Integration.NewConnectedMode
+{
+    internal class UnbindCommand
+    {
+        private readonly IHost host;
+        private readonly IConfigurationProvider configProvider;
+
+        public UnbindCommand(IHost host)
```

Comment:
I considered using a similar pattern to the existing binding workflow/executor/controller, but it seemed unnecessarily complicated for the unbind case. Similarly, the unbind should be fast enough not to need the progress bar code.
The _Execute_ expects to be on a UI thread so I've added an assertion for that.